README.md

```diff
@@ -0,0 +1,28 @@
+## mattermost-plugin-topdf
+_TOPDF_ plugin converts Office files that attached to posts to PDF files and caches them for future use. To convert a file to PDF simply make a `GET` request to:
+```
+http://{mattermostAddr}/plugins/topdf/files/{fileID}
+```
+
+This plugin also enables previewing Office files on _webapp_.
+
+## Installation
+1. Go to the [releases page of this Github repository](https://github.com/ilgooz/mattermost-plugin-topdf/releases) and download the latest release for your Mattermost server.
+   
+2. In the Mattermost System Console under **System Console > Plugins > Plugin Management** upload the file to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+
+3. Install _Gotenberg_ server since this plugin depends on it to actually convert files to PDFs. Assuming you have already installed Docker, following command will download and install the latest version of _Gotenberg_ and set up a server with an open port at 4798:
```

Comment:
I wanted to point out that Gotenberg server needs custom timeouts since creating PDFs might take some time and defaults are too low.

I also dropped a Docker command there that includes correct timeouts so this way I think it's a bit easier for users to just copy-paste that command and run the server.

But I'll add the installation link too.

done by b16f1afa1fef546a7dcce06e8df55c0472e5b539 and 78c470413a966f1ddd00f8a0d0b165d7d138379a